test/test-functions

```diff
@@ -724,6 +724,16 @@ setup_basic_environment() {
     if get_bool "$TEST_INSTALL_VERITY_MINIMAL"; then
         install_verity_minimal
     fi
+
+    # Send stdout/stderr of testsuite-*.service units to both journal and
+    # console to make debugging in CIs easier
+    # Note: we can't use a dropin for `testsuite-.service`, since that also
+    #       overrides 'sub-units' of some tests that already use a specific
+    #       value for Standard(Output|Error)=
+    #       (e.g. test/units/testsuite-66-deviceisolation.service)
+    local dropin_dir="${initdir:?}/etc/systemd/system/testsuite-${TESTID:?}.service.d"
+    mkdir -p "$dropin_dir"
+    printf '[Service]\nStandardOutput=journal+console\nStandardError=journal+console' >"$dropin_dir/99-stdout.conf"
```

Comment:
I'm not sure it should be turned on by default in interactive mode. I think it would make sense to wrap it in `[[ ! get_bool "$INTERACTIVE_DEBUG" ]]` or something like that.

Reply:
Good point! Fixed.